src/builtins.cpp

```diff
@@ -173,7 +174,7 @@ static bool lCreateISPCSymbol(llvm::Function *func, SymbolTable *symbolTable) {
     // symbol creation code below assumes that any LLVM vector of i32s is a
     // varying int32.  Here, we need that to be interpreted as a varying
     // bool, so just have a one-off override for that one...
-    if (g->target->getMaskBitCount() != 1 && name == "__sext_varying_bool") {
+    if (g->target->getMaskBitCount() != 1 && name == __sext_varying_bool) {
```

Comment:
In despite that you have `using namespace ispc::builtin;`, I would write `builtin::__sext_varying_bool` here for better readability.

Reply:
Totally agree.

Reply:
I've fixed that